.docs/Using-the-GUI.md

```diff
@@ -0,0 +1,97 @@
+# Using the GUI
+
+## Video tutorial
+
+[![Video tutorial](https://i.ytimg.com/vi/jjtu0VQXV7I/hqdefault.jpg)](https://youtube.com/watch?v=jjtu0VQXV7I)
+
+> Video by [NoIntro Tutorials](https://youtube.com/channel/UCFezKSxdNKJe77-hYiuXu3Q).
+
+## Guide
+
+### Step 1
+
+After extracting the `.zip`, open `DiscordChatExporter.exe`
+
+### Step 2
+
+Please refer to the on-screen instructions to get your token, then paste your token in the upper text box and hit ENTER or click the arrow (→).
+
+> **Warning**:
+> **Never share your token!**
+> A token gives full access to an account, treat it like a password.
+
+<img src="https://i.imgur.com/SuLQ5tZ.png" height="400"/>
```

Comment:
Do we need to control the `height` here or can we get away with a normal markdown image markup?

Reply:
It gets way too big with a normal markdown image.